wordpress-mobile/WordPress-iOS
# Remove the old-style app icon

I realized the old App Icon is still present – I don't think we need it anymore with the new Muriel colour style? 

**To Test:**
- Ensure that CI tests are green before starting.
- Ensure that the main icon works correctly.
- Ensure that alternate icons work correctly.
- Build and **archive** the `WordPress Internal` target. Find the archive in the organizer, choose "Show in Finder", and see that the banner is present (it should be `13.1.0.12190814`)
